support/src/testFixtures/java/se/inera/intyg/common/support/facade/testsetup/model/config/ConfigTest.java

```diff
@@ -88,11 +88,9 @@ void shouldIncludeIcon() {
     @Test
     void shouldIncludeConfigText() {
         final var question = getElement();
-        if (getTextId() != null && !question.getConfig().getText().equals(getTextId())) {
+        if (getTextProviderMock() != null && getTextId() != null && !question.getConfig().getText().equals(getTextId())) {
             assertTrue(question.getConfig().getText().trim().length() > 0, "Missing text");
             verify(getTextProviderMock(), atLeastOnce()).get(getTextId());
-        } else {
-            assertEquals(getTextId(), question.getConfig().getText());
         }
```

Comment:
If I am thinking correctly, by removing the else clause here we miss cases where the returned value is null but the test is expecting something else...

Reply:
Ive taken back the else statement and instead changed the mocks so that the desired textId is returned.